src/logic/lint.test.ts

```diff
@@ -16,6 +15,7 @@ import {
   ValidationResult,
 } from '@superfaceai/parser';
 import { SyntaxErrorCategory } from '@superfaceai/parser/dist/language/error';
+import { MatchAttempts } from '@superfaceai/parser/dist/language/syntax/rule';
```

Comment:
why are these imports like this?

Reply:
Because parser has weird exports 😩

Reply:
Should we fix this in Parser? This only effects the tests🤷